src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewDeveloperSteps.java

```diff
@@ -0,0 +1,80 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import gov.healthit.chpl.aqa.pageObjects.ViewDeveloperPage;
+
+/**
+ * Class ViewDeveloperSteps definition.
+ */
+public class ViewDeveloperSteps extends Base {
+
+    /**
+     * Get user to the View Developer Page.
+     * @param developerId is Developer ID
+     **/
+    @Given("^I navigate to the developer edit page for developer \"([^\"]*)\"$")
```

Comment:
This isn't navigating to the edit page. This is navigating to the developer page. If you want to navigate to the edit page then the URL would be `#/organizations/developers/{ developerID }/edit`. Should either change the step definition or change the URL